### PR TITLE
Minimum change to test Python 3

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by rabbah** [comment]
see #2044.

File: core/actionProxy/actionproxy.py [hunk 1]
```diff
@@ -251,6 +250,7 @@ def error():
 
 def complete(response):
     # Add sentinel to stdout/stderr
+    print(response)
```
Comment (rabbah) [inline core/actionProxy/actionproxy.py]: Extra logging may cause tests to fail (which might be asserting empty logs ie no message from proxy appearing in action logs).

Comment (cclauss) [inline core/actionProxy/actionproxy.py]: Got it.  Thanks.

**Comment by cclauss** [comment]
Closing as no longer needed.  #2044 contained the solution.  Results will be bundled into #1903.
